tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -160,6 +160,10 @@ class TrtConversionParams(
       inputs during runtime, then a new TensorRT engine is built at runtime if
       allow_build_at_runtime=True, and otherwise native TF is used. This
       argument is only effective if is_dynamic_op=True.
+    allow_mixed_precision_on_native_ops: whether to allow TensorFlow to
```

Comment:
I would call this parameter `allow_low_precision_on_native_ops`, since the phrase "mixed precision" is not used in the documentation for `precision_mode`. Also, I wouldn't mention AMP directly, as that is an implementation detail. Instead you can state something like "whether to allow ops to be converted to lower precision if they are not run by TensorRT."

Reply:
@reedwm @bixia1  : I don't like the idea of renaming `mixed_precision` to `low_precision`. It's poorly accurate and could rapidly be confusing, what does it mean:  FP16 only ? BF16 only ? AMP ? INT8 ?

TF-TRT gives you fine control on which precision you requested: FP32 / FP16 / INT8. I think we should try to be specific and have an explicit API. I don't see any reason to make this API more fuzzy/blackbox

Reply:
Fair point, I agree it's fine to leave it as `mixed_precision`. Still, I don't think you should refer to AMP specifically, but instead just refer to the fact ops will be converted to use mixed precision.

Reply:
hmm, the grappler pass auto_mixed_precision only convert FP32 to FP16 or BF16. So I think low_precision is more proper.

Reply:
@bixia1 the reasoning is that I want to be specific to what "low precision" means. There has been no marketing effort spent "low_precision", nobody will know what's behind. On the other hand, "(automatic) mixed_precision" is a well publicized concept in the deep learning community and now TF users start to be well familiarized with the idea. I don't think it's a smart move to change the naming of a well established and identified concept/mechanism.